nixos/modules/services/logging/logrotate.nix

```diff
@@ -6,6 +6,8 @@ let
   cfg = config.services.logrotate;
 
   # deprecated legacy compat settings
+  # these options will be removed before 22.11 in the following PR:
+  # https://github.com/NixOS/nixpkgs/pull/164169k
```

Comment:
there's a k

Reply:
should be https://github.com/NixOS/nixpkgs/pull/164169

Reply:
ugh, sorry - fixed in a force push -_-